source/styles/layout/_header.scss

```diff
@@ -0,0 +1,44 @@
+.header-block {
+	display: flex;
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	z-index: 1;
+	&__nav {
+		margin-top: 21px;
+		width: 100%;
+		z-index: 1;
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		.nav__logo {
+			max-width: 113px;
+			&__img {
+				width: 100%;
+				height: 67px;
+			}
+		}
```

Comment:
To już nie jest bem, stwórz component dla samego  logo
```
<span class="header__logo header-logo">
      <a href="/" class="header-logo__link">
        {% include '../../assets/svg/logo.svg' %}
      </a>
    </span> 
```
z czego header__logo nadaje pozycje jeśli jest potrzebna a  header-logo styl